documentation/pom.xml

```diff
@@ -22,7 +22,15 @@
     <description>The Hibernate Validator reference manual</description>
 
     <properties>
-        <jdocbook.ignoreTranslations>false</jdocbook.ignoreTranslations>
+        <asciidoctor.base-outputdir>${basedir}/target/asciidoctor</asciidoctor.base-outputdir>
+        <!--
+        With the asciidoctor Maven plugin 1.6, we are going to put back the images into the src/main/asciidoc
```

Comment:
See the commented part with resources. Asciidoctor currently copies all the resources from src/main/asciidoc so you end up with all the images/css/docinfo files, even in the PDF or epub case. 1.6 provides a resources option allowing to filter the elements copied.

I'll make that more clear.